weblate_web/templates/snippets/reviews.html

```diff
@@ -0,0 +1,384 @@
+{% load i18n %}
+{% load static %}
+
+<style>
+
+.testimonial1{
+    width: 365px;
+    height: 347px;
+    border-radius: 30px;
+    background-color: #ffffff;
+    box-shadow: 0 2px 10px 0 rgba(0,0,0,0.1);
+    margin-top: 10px;
+}
+
+.testimonial2{
+    height: 291px;
+    width: 365px;
+    border-radius: 30px;
+    background-color: #FFFFFF;
+    box-shadow: 0 2px 10px 0 rgba(0,0,0,0.1);
+    margin-top: 33px;
+}
+
+.testimonial3{
+    width: 365px;
+    height: 347px;
+    border-radius: 30px;
+    background-color: #ffffff;
+    box-shadow: 0 2px 10px 0 rgba(0,0,0,0.1);
+    margin-top: 10px;
+}
+
+.icon-review{
+    text-align: left;   
+}
```

Comment:
This will most likely need a variant for RTL languages (there are more such cases in the CSS).

Reply:
It has been done!